packages/perseus/src/widgets/__tests__/interactive-graph.test.ts

```diff
@@ -527,4 +528,34 @@ describe("locked layer", () => {
             stroke: lockedFigureColors.pink,
         });
     });
+
+    test("should render locked circles", async () => {
+        // Arrange
+        const {container} = renderQuestion(segmentWithLockedCircles, {
+            flags: {
+                mafs: {
+                    segment: true,
+                },
```

Comment:
Same comment as earlier about using flagsForApiOptions.

I had another thought though.  Should these new locked figures be behind an M2 feature flag (like we [had for M1](https://github.com/Khan/perseus/blob/main/packages/perseus-editor/src/__stories__/flags-for-api-options.ts#L7))?

Maybe work that out with Mark.  I could see us shipping each locked figure as they're ready and tested, especially given we talked to Sarah P about possibly not needing full QE passes on locked figures.